src/main/java/Tasks/Task.java

```diff
@@ -0,0 +1,51 @@
+package Tasks;
+
+/**
+ * The Task class represents a generic task.
+ */
+public class Task {
```

Comment:
Have you considered making the class abstract? From the samples given in later levels it seems like all tasks need a type.